### DISCUSSION TITLE
Misc

### DISCUSSION THREAD

**Discussion by sgamesdev** [Leo-Corporation/Gavilya#285] [General]
That's why c++ is more popular that c#

**Comment by lpeyr** [comment]
Yes, you have more control over your code.

**Reply by sgamesdev** [reply]
You should obsfucate the .dll , just run dotObsufucator , it makes the reverse engeneering a lit bit harder , your api key is ending in 3118

**Reply by lpeyr** [reply]
Ok, will do that in the next release.
